src/scripting/lua_map.cc

```diff
@@ -2747,11 +2793,9 @@ ProductionSiteDescription
    Child of: :class:`MapObjectDescription`, :class:`ImmovableDescription`,
    :class:`BuildingDescription`
 
-   A static description of a tribe's productionsite, so it can be used in help files
-   without having to access an actual building on the map.
-   This class contains the properties for productionsites that have workers.
-   For militarysites and trainingsites, please use the subclasses.
-   See the parent classes for more properties.
+   A static description of a tribe's productionsite. This class contains
+   the properties for productionsites that have workers. For militarysites and trainingsites,
```

Comment:
trainingsites are also "productionsites that have workers", and militarysites are not productionsites at all

Reply:
removed the sentence

Reply:
The other sentence can also go as all productionsites need a worker